rules/test.bzl

```diff
@@ -105,6 +105,7 @@ def _ios_test(name, test_rule, test_suite_rule, apple_library, infoplists_by_bui
             tests.append(test_name)
             split_rule(
                 name = test_name,
+                module_name = kwargs["module_name"],
```

Comment:
Would adding `"module_name"` to `_IOS_TEST_KWARGS` accomplish the same thing while keeping all the important keys in one place?

Reply:
Yeah it should. Tried it at first but it failed for a way i didn't understand and din't have to time to debug. I actually didn't mean to push this up but was int he wrong repo :). I need to debug why it wasn't working and hopefully get it to work

Reply:
So this wasn't able to be done because the kwargs["module_name"] was also used in `apple_library` our current implementation of `_IOS_TEST_KWARGS` pops them off so they aren't passed into apple_library. I added a comment explaining in code so it's not as confusing int he future